GitUI/GitUICommands.cs

```diff
@@ -1725,8 +1725,22 @@ private bool RunFileHistoryCommand(IReadOnlyList<string> args, bool showBlame)
                 filterByRevision = true;
             }
 
-            ShowModelessForm(owner: null, requiresValidWorkingDir: true, preEvent: null, postEvent: null,
+            // Similar to StartFileHistoryDialog(), tests works better with this setup
```

Comment:
Did you mean this?
```suggestion
            // Similar to StartFileHistoryDialog(), tests work better with this setup
```

Reply:
Removing second part, this should not be launched but be opened as a modeles form